// @test error

fn main() : int
{
	2 + true;
	return 20;
}